src/testing/jest/jest-27-and-under/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
See the commit body where I added this file for details behind some of the decisions. Happy to answer questions about this one